src/CMakeLists.txt

```diff
@@ -483,6 +483,10 @@ target_include_directories(${target} PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/layouts>
     $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/nodes>)
 
+target_include_directories(${target}
+    INTERFACE
+        $<INSTALL_INTERFACE:${QSK_INSTALL_HEADERS}>)
```

Comment:
@uwerat this line makes it that cmake can automatically generate the (library) target's include directories